### PR TITLE
Implement part of backend

### PR DESCRIPTION
**Pull request by Soroyesodiq**
Implement some basic functionality like the calculate button.

### CODE REVIEW

File: app/src/main/java/com/example/medapp/MainActivity.java [hunk 1]
```diff
@@ -2,33 +2,131 @@
...
+                DisplayResult(totalScore);
+
+            }
+        });
+
+    }
+    //This display the heart failure percentage, summary and advise.
+    //An AlertBuilder is used for the display
+    private void DisplayResult(int result) {
+        String summaryMessage = "";
```
Comment (teslimSulaiman) [inline app/src/main/java/com/example/medapp/MainActivity.java:82]: clean up this logic, you can create a nested object or map. presently it is not visible. You can move it to a viewModel or presenter.

**Review by teslimSulaiman — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/medapp/MainActivity.java (hunk 1 above)
Comment (Soroyesodiq) [reply app/src/main/java/com/example/medapp/MainActivity.java:82]: Can you please explain further. I don't understand very well.

Comment (teslimSulaiman) [reply app/src/main/java/com/example/medapp/MainActivity.java:82]: business logic should not be in the view, move it to a presenter. check MVP or use viewModel MVVM

Comment (Soroyesodiq) [reply app/src/main/java/com/example/medapp/MainActivity.java:82]: Okay, I will check that out

**Pull request merged**